src/main/java/Todo.java

```diff
@@ -0,0 +1,11 @@
+public class Todo extends Task{
```

Comment:
Perhaps `Task` and its proceeding `{` could have a space in between them?

Reply:
Alrighty!